README.md

```diff
@@ -69,6 +70,13 @@ Default color is #FFFFFF (White). Make sure this color is a hex value.
 
 ![image](https://user-images.githubusercontent.com/45357714/131814904-4ae06504-8dc2-4ba0-8b02-e4bba79faf80.png)
 
+### Customizable Acronyms
```

Comment:
Could you please also document these new switches in #convert-file and #convert-url.